pool/app/pool_database/pool_database.ml

```diff
@@ -6,6 +6,25 @@ let src = Logs.Src.create "database"
 
 module Logs = (val Logs.src_log src : Logs.LOG)
 
+let connect_and_migrate database_label =
```

Comment:
I propose to update the naming a little to make clear this function in only supposed to handle migrations with connection issues. Or it could be integrated in the `rerun_migrations_for_connection_issues` function.

```suggestion
let reconnect_and_migrate database_label =
```